tests/platform_tests/api/test_chassis_fans.py

```diff
@@ -221,3 +221,17 @@ def test_set_fans_led(self, duthost, localhost, platform_api_conn):
                             color, color_actual, i))
 
         self.assert_expectations()
+
+    def test_get_position_in_parent(self, platform_api_conn):
+        for i in range(self.num_fans):
+            position = fan.get_position_in_parent(platform_api_conn, i)
+            self.expect(position is not None, "Failed to perform get_position_in_parent for fan {}".format(i))
+            self.expect(isinstance(position, int), "Position value must be an integer value for fan {}".format(i))
```

Comment:
The second `self.expect()` call is unnecessary if the first fails, so it should be nested using `if self.expect()` on the first. See lines 213-214 as an example.